exporter/exporterhelper/README.md

```diff
@@ -24,6 +24,8 @@ The following configuration options can be modified:
       [the batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor)
       is used, the metric `batch_send_size` can be used for estimation)
 - `timeout` (default = 5s): Time to wait per individual attempt to send data to a backend
+- `sampled_logger`: Enables/disables sampling of logging messages.
```

Comment:
It is not clear what does enabling/disabling sampling do. Please expand the description.

Reply:
@tigrannajaryan thank you for your feedback.
Is it now more clear? Am I missing something?

Thanks